src/main/java/racingcar/step3/Race.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.step3;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Race {
+
+    private List<Car> cars = new ArrayList<>();
```

Comment:
```suggestion
    private final List<Car> cars = new ArrayList<>();
```
cars 변수에 할당된 리스트가 재할당될 수 없도록 final을 붙여주면 좋을 것 같네요